charts/csc_shared/templates/configmap-env.yaml

```diff
@@ -4,6 +4,7 @@ kind: ConfigMap
 metadata:
   name: csc-env-config
 data:
+    EXTERNAL_INSTANCE_URL: {{ .Values.global.baseUrl | quote }}
```

Comment:
``global.baseUrl`` is constructed using ``quote``, so this second usage is redundant.